packages/common/src/hooks/chats/useChatBlastAudienceContent.ts

```diff
@@ -37,23 +37,23 @@ export const useChatBlastAudienceContent = ({ chat }: { chat: ChatBlast }) => {
     enabled: audienceContentType === 'album'
   })
 
-  const { data: purchasersCount } = useGetPurchasersCount(
+  const { data: purchasersCount } = usePurchasersCount(
     {
       userId: currentUserId!,
       contentId: decodedContentId,
       contentType: audienceContentType
     },
     {
-      disabled: audience !== ChatBlastAudience.CUSTOMERS || !currentUserId
+      enabled: !!(audience === ChatBlastAudience.CUSTOMERS && currentUserId)
```

Comment:
You can omit `currentUserId` here since we check the userId arg inside the tan-query hook